### PR TITLE
Make sure the image URL is returned when a replied message is deleted

### PR DESCRIPTION
**Pull request by cammellos**
When we reply to our own message with an image, we didn't set the URL of the image of the message, which resulted in the image not being displayed correctly.

### CODE REVIEW

**Review by qfrank — APPROVED** [review]
LGTM

File: protocol/messenger_delete_message_test.go [hunk 1]
```diff
@@ -518,3 +478,58 @@ func (s *MessengerDeleteMessageSuite) TestDeleteMessageAndChatIsAlreadyRead() {
...
+	// create an http server
+	mediaServer, err := server.NewMediaServer(nil, nil, nil)
+	s.Require().NoError(err)
+	s.Require().NotNil(mediaServer)
+	s.Require().NoError(mediaServer.Start())
+
+	theirMessenger.httpServer = mediaServer
+
+	// We reply to our own message with an image
+	image, err := buildImageWithoutAlbumIDMessage(*theirChat)
```
Comment (qfrank) [inline protocol/messenger_delete_message_test.go]: oh, can we rename `image` to `imageMessage`?

Comment (cammellos) [reply protocol/messenger_delete_message_test.go]: good point

**Review by jrainville — APPROVED** [review]

File: protocol/messenger_messages.go [hunk 2]
```diff
@@ -229,7 +229,9 @@ func (m *Messenger) DeleteMessageAndSend(ctx context.Context, messageID string)
 		if err != nil {
 			return nil, err
 		}
+
 		response.AddMessages(updatedMessages)
+		m.prepareMessages(response.messages)
```
Comment (Samyoul) [inline protocol/messenger_messages.go:234]: That's all it took? 😆

**Review by Samyoul — APPROVED** [review]

**Pull request merged**